pkg/operator/apiserver/controller/workload/workload.go

```diff
@@ -161,40 +164,33 @@ func (c *Controller) updateOperatorStatus(ctx context.Context, previousStatus *o
 		errs = []error{}
 	}
 
-	deploymentAvailableCondition := operatorv1.OperatorCondition{
-		Type:   fmt.Sprintf("%sDeployment%s", c.conditionsPrefix, operatorv1.OperatorStatusTypeAvailable),
-		Status: operatorv1.ConditionTrue,
-	}
+	deploymentAvailableCondition := applyoperatorv1.OperatorCondition().
+		WithType(fmt.Sprintf("%sDeployment%s", c.conditionsPrefix, operatorv1.OperatorStatusTypeAvailable)).
+		WithStatus(operatorv1.ConditionTrue)
```

Comment:
Thought we agreed to initialize without status to ensure that every branch sets all conditions

Reply:
My bad - fixed.